### PR TITLE
[SFT-586]: form attributes

### PR DESCRIPTION
**Pull request by gustavs-gutmanis**
* adding logic for passing field attributes through form attributes

[Documentation](https://solspace.atlassian.net/wiki/spaces/SOFTWARE/pages/110624769/Twig+Function+parameters)

### CODE REVIEW

File: packages/plugin/src/Freeform.php [hunk 1]
```diff
@@ -342,6 +342,7 @@ private function initServices(): void
                 'files' => FilesService::class,
                 'forms' => FormsService::class,
                 'field' => FieldsService::class,
+                'fields' => FieldsService::class,
```
Comment (seandelaney) [inline packages/plugin/src/Freeform.php:345]: Are these both meant to be `FieldsService` ?

Comment (gustavs-gutmanis) [reply packages/plugin/src/Freeform.php:345]: I thought I removed one of them, thanks for noticing!

**Review by seandelaney — APPROVED** [review]
I cant seem to replicate Kelseys issue

**Review by kjmartens — APPROVED** [review]
Thoroughly tested and updated sample formatting templates. Noticed a couple of minor issues but I'll make a separate task for those

**Pull request merged**